includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -186,6 +191,7 @@ public function get_settings(): WP_REST_Response {
 				'payment_request_button_size'       => $this->wcpay_gateway->get_option( 'payment_request_button_size' ),
 				'payment_request_button_type'       => $this->wcpay_gateway->get_option( 'payment_request_button_type' ),
 				'payment_request_button_theme'      => $this->wcpay_gateway->get_option( 'payment_request_button_theme' ),
+				'is_saved_cards_enabled'            => 'yes' === $this->wcpay_gateway->get_option( 'saved_cards' ),
```

Comment:
You can use `$this->is_saved_cards_enabled()`. It is being used by other places in the code. You can also argue that less wrapper function is better. :)

Reply:
Thanks for pointing that out, I didn't notice the method - updating 👍 
```suggestion
				'is_saved_cards_enabled'            => $this->wcpay_gateway->is_saved_cards_enabled(),
```